### PR TITLE
[FR] Add `Skinning/osu!`, `Skinning/osu!catch`

### PR DESCRIPTION
**Pull request by DoryanR**
Translation of the following wiki pages:

- `Skinning/osu!`
- `Skinning/osu!catch`
- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

*Feel free to merge if the delay is too long for a review by a native.*

### CODE REVIEW

File: wiki/Skinning/osu!/fr.md [hunk 1]
```diff
@@ -0,0 +1,765 @@
...
+![](img/spinner-glow.png)
+
+| Versions | Peut être animé | Modifiable sur une beatmap | Blend Mode | Origine | Taille de SD suggérée |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| 2.0+ | ![Non][false] | ![Oui][true] | Additif | Centre | - |
+
+Notes :
+
+- Cet élément est positionné à environ 397px verticalement.
+- Teinté en cyan, clignote en blanc lorsque des points bonus sont attribués.
```
Comment (Ganondorf-1) [inline wiki/Skinning/osu!/fr.md:652]:
```suggestion
- Teinté de cyan, clignote en blanc lorsque des points bonus sont attribués.
```

File: wiki/Skinning/osu!catch/fr.md [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+
+| Versions | Peut être animé | Modifiable sur une beatmap | Blend Mode | Origine | Taille de SD suggérée |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| 2.3+ | ![Oui][true] | ![Oui][true] | Normal | Centre | - |
+
+Notes:
+
+- Nom de l'animation : `fruit-catcher-fail-{n}.png`
+- Cet élément est l'état "manqué" de l'attrapeur.
+- Ceci remplace `fruit-catcher-kiai.png` si un fruit ou un drop/droplet est manquée pendant le [kiai time](/wiki/Kiai_time).
```
Comment (Ganondorf-1) [inline wiki/Skinning/osu!catch/fr.md:37]:
```suggestion
- Ceci remplace `fruit-catcher-kiai.png` si un fruit ou un drop/droplet est manqué pendant le [kiai time](/wiki/Kiai_time).
```

File: wiki/Skinning/osu!catch/fr.md [hunk 3]
```diff
@@ -0,0 +1,297 @@
...
+
+`fruit-drop-overlay.png`
+
+| Versions | Peut être animé | Modifiable sur une beatmap | Blend Mode | Origine | Taille de SD suggérée |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| All | ![Non][false] | ![Oui][true] | Normal | Centre | 128x128 |
+
+Notes:
+
+- Cet élément n'est pas utilisé dans l'écran de classement.
```
Comment (Ganondorf-1) [inline wiki/Skinning/osu!catch/fr.md:293]:
```suggestion
- Cet élément n'est pas utilisé dans l'écran de résultats.
```

File: wiki/Skinning/osu!/fr.md [hunk 4]
```diff
@@ -0,0 +1,765 @@
...
+
+| Versions | Peut être animé | Modifiable sur une beatmap | Blend Mode | Origine | Taille de SD suggérée |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| Toutes | ![Oui][true] (voir notes) | ![Oui][true] | Normal | Centre | 118x118 (cercle) 128x128 (fichier entier) |
+
+Notes :
+
+- Nom de l'animation : `sliderstartcircleoverlay-{n}.png`
+- Taux d'animation : 2 FPS (4 FPS max).
+  - Ce taux est affecté par les modificateurs de mi-temps et de double temps/nightcore du jeu.
```
Comment (Ganondorf-1) [inline wiki/Skinning/osu!/fr.md:331]:
```suggestion
  - Ce taux est affecté par les mods Half Time et Double Time/Nightcore.
```

File: wiki/Skinning/osu!/fr.md [hunk 5]
```diff
@@ -0,0 +1,765 @@
...
+
+| Versions | Peut être animé | Modifiable sur une beatmap | Blend Mode | Origine | Taille de SD suggérée |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| Toutes | ![Oui][true] (voir notes) | ![Oui][true] | Normal | Centre | 118x118 (cercle) 128x128 (fichier entier) |
+
+Notes :
+
+- Nom de l'animation : `hitcircleoverlay-{n}.png`
+- Fréquence d'animation : 2 FPS (4 FPS max).
+  - Ce taux est affecté par les modificateurs half time et double time/nightcore du jeu.
```
Comment (Ganondorf-1) [inline wiki/Skinning/osu!/fr.md:244]:
```suggestion
  - Ce taux est affecté par les mods Half Time et Double Time/Nightcore.
```

File: wiki/Skinning/osu!catch/fr.md [hunk 6]
```diff
@@ -0,0 +1,297 @@
+# Skinning du mode osu!catch
+
+*Voir également : [Tutoriel de skinning  du mode osu!catch](/wiki/Skinning/Guides_and_important_threads)*
```
Comment (Ganondorf-1) [inline wiki/Skinning/osu!catch/fr.md:3]:
```suggestion
*Voir également : [Tutoriel de skinning du mode osu!catch](/wiki/Skinning/Guides_and_important_threads)*
```

File: wiki/Skinning/osu!catch/fr.md [hunk 7]
```diff
@@ -0,0 +1,297 @@
...
+
+| Versions | Peut être animé | Modifiable sur une beatmap | Blend Mode | Origine | Taille de SD suggérée |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| 2.3+ | ![Oui][true] | ![Oui][true] | Normal | Centre | - |
+
+Notes:
+
+- Nom de l'animation : `fruit-catcher-kiai-{n}.png`
+- Cet élément est l'état de l'attrapeur pendant le kiai time.
+- `fruit-catcher-fail.png` remplacera ceci si un fruit ou un drop/droplet est manquée pendant le [kiai time](/wiki/Kiai_time).
```
Comment (Ganondorf-1) [inline wiki/Skinning/osu!catch/fr.md:54]:
```suggestion
- `fruit-catcher-fail.png` remplacera ceci si un fruit ou un drop/droplet est manqué pendant le [kiai time](/wiki/Kiai_time).
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

**Comment by DoryanR** [comment]
@Ganondorf-1 recheck please

**Review by TicClick — APPROVED** [review]
merging with the assumption that one review was enough

**Pull request merged**